homeassistant/components/emoncms/const.py

```diff
@@ -0,0 +1,18 @@
+"""Emoncms constants."""
+
+import logging
+
+ATTR_FEEDID = "FeedId"
+ATTR_FEEDNAME = "FeedName"
+ATTR_LASTUPDATETIME = "LastUpdated"
+ATTR_LASTUPDATETIMESTR = "LastUpdatedStr"
+ATTR_SIZE = "Size"
+ATTR_TAG = "Tag"
+ATTR_USERID = "UserId"
+CONF_EXCLUDE_FEEDID = "exclude_feed_id"
+CONF_ONLY_INCLUDE_FEEDID = "include_only_feed_id"
+CONF_SENSOR_NAMES = "sensor_names"
+DECIMALS = 2
+ONLY_INCL_EXCL_NONE = "only_include_exclude_or_none"
+
+LOGGER = logging.getLogger(__name__)
```

Comment:
I think it makes more sense in that case to use
```suggestion
LOGGER = logging.getLogger(__package__)
```